indico/modules/events/registration/clone.py

```diff
@@ -22,48 +22,88 @@
 from indico.modules.events.registration.models.forms import RegistrationForm
 from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
 from indico.modules.events.registration.models.items import RegistrationFormItem, RegistrationFormSection
+from indico.modules.events.registration.models.registrations import Registration, RegistrationData
 from indico.util.i18n import _
 
 from MaKaC.conference import EventCloner
 
 
 class RegistrationFormCloner(EventCloner):
+    def __init__(self, *args, **kwargs):
+        super(RegistrationFormCloner, self).__init__(*args, **kwargs)
+        self._field_data_dict = {}
```

Comment:
I'd call it `_field_data_map`. It's a mapping. The fact that it's a dict is not really something that belong in the name (reminds me of this terrible thing called hungarian notation)